cmd/operator-sdk/new/cmd.go

```diff
@@ -354,14 +353,14 @@ func doHelmScaffold() error {
 	}
 
 	if err := scaffold.UpdateRoleForResource(resource, cfg.AbsProjectPath); err != nil {
-		return fmt.Errorf("failed to update the RBAC manifest for resource (%v, %v): %w", resource.APIVersion, resource.Kind, err)
+		return fmt.Errorf("failed to update the RBAC manifest for resource (%v, %w): %w", resource.APIVersion, resource.Kind, err)
```

Comment:
```suggestion
		return fmt.Errorf("failed to update the RBAC manifest for resource (%v, %v): %w", resource.APIVersion, resource.Kind, err)
```

Reply:
Sorry. My bad. Changed.

Reply:
This file is also under `cmd/operator-sdk`. So shall I remove %w here as per [this comment](https://github.com/operator-framework/operator-sdk/pull/2355/files#r362048008), OR there are certain files under `cmd/operator-sdk` that needs to be specifically looked into.

Reply:
Ignore this comment and remove `%w` changes as specified [here](https://github.com/operator-framework/operator-sdk/pull/2355#discussion_r362048008).

Reply:
Done.